### PR TITLE
mrc-670: Create staging instance of hint

### PR DESCRIPTION
**Pull request by richfitz**
This PR creates a staging copy of hint, using the usual pattern of a Vagrant-configured VM, which is now running on naomi.dide.ic.ac.uk

* There is no proxy yet (#2, but this depends on how mrc-666 resolves)
* The port will clash with the port used by the real copy, but we'll hold of on fixing that until we know if it is us or ICT doing the proxying

I'll add ssh keys to the staging machine.

### CODE REVIEW

File: staging/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Vagrant.require_version ">= 1.8.2"
+
+domain = 'localdomain'
+box = "bento/ubuntu-18.04"
+ram_size_mb = '16384'
+
+# vault_config_file = 'shared/vault_config'
```
Comment (EmmaLRussell) [inline staging/Vagrantfile]: These commented lines are obsolete?

File: staging/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+```
+ssh hint@naomi.dide.ic.ac.uk
+```
+
+(if you need sudo on this machine, the password is in the vault (`vault.dide.ic.ac.uk:8200`) as `/secret/hint/server/login`
+
+ (ebola2018.dide.ic.ac.uk), run the following commands.
+
+```
+git clone https://github.com/imperialebola2018/hint-deploy
```
Comment (EmmaLRussell) [inline staging/README.md]: imperialebola2018 rather than mrc-ide?

File: staging/Vagrantfile (hunk 1 above)
Comment (richfitz) [reply staging/Vagrantfile]: yeah - I was thinking we'd put them back in soon but am not currently that sure that they're needed...

**Comment by richfitz** [comment]
Please leave this one open now until mrc-665 / #2 is resolved as we'll need to use the cert there

**Comment by richfitz** [comment]
ok, this one is good to go now

**Pull request merged**